### PR TITLE
Use docker mounts to include user source code

### PR DESCRIPTION
**Pull request by frthjf**
This allows users to dynamically mount user source code like the example cases into the docker container.

### CODE REVIEW

File: docs/tutorial/docker.md [hunk 1]
```diff
@@ -23,44 +23,72 @@ flowchart LR
...
+The command below both _initialize_ a container `miv_env:0.1` and _launch_ that container. By default, the container will run the `jupyter-lab` in the background on the port `8888`. User can alther the destination port by setting `<destination port>:8888`.
 
 ```bash
-docker run -p 8888:8888 -it miv_env:0.1  # Launch jupyter-lab
+docker run -p 8888:8888 -it miv_env:0.1
 ```
 
-The above command both _initialize_ a container `miv_env:0.1` and _launch_ that container. By default, the container will run the `jupyter-lab` in the background on the port `8888`. User can alther the destination port by setting `<destination port>:8888`.
-
 To see the running containers, run `docker ps` or `docker container ls`. Too show all containers including the one stopped, pass `-a` or `-all`.
```
Comment (frthjf) [inline docs/tutorial/docker.md:58]: Just an observation: this feels like scope creep since the documentation shouldn't be the place to teach generic docker skills. My suggestion would be to leave the tailored version of the commands as they apply to MiV and send people to the official documentation for anything more generic.

**Comment by skim0119** [comment]
Thanks! I'll merge it once the docker-build completes.

File: docs/tutorial/docker.md (hunk 1 above)
Comment (skim0119) [reply docs/tutorial/docker.md:58]: I agree that I should be careful, but I would like to include enough information to run the docker in our MiV-Simulator context without digging into the docker much. I'll remove technical descriptions.

**Comment by frthjf** [comment]
Sounds good, thanks!

**Review by skim0119 — APPROVED** [review]
